backend/layers/business/business.py

```diff
@@ -972,6 +972,18 @@ def publish_collection_version(
             if canonical_datasets != version_datasets:
                 has_dataset_revisions = True
 
+            # Update publisher metadata if there are updates in Crossref that have occurred since last publish.
+            doi_link = next((link for link in version.metadata.links if link.type == "DOI"), None)
```

Comment:
For my own understanding, this returns the first value in the iterator that matches `if link.type == "DOI"` or `None`?

Reply:
That's right!